Nmaksimchicov/2.HTML/Задание 3.html

```diff
@@ -0,0 +1,73 @@
+ 
+<pre>#======================================= 
+# random_example.py # Module example using the random module
+#=======================================
+import random
+# print 14 random integers
+for cntr in range(1,15):    
+	print random.randint(1,10)</pre>
+<p>Давайте разберём код по строкам. Первые четыре строки — это комментарии, о которых мы говорили в прошлый раз. Пятая строка говорит Python, что нужно использовать модуль «<em>random</em>». Мы должны явно указать Python на это.</p>
+
+<hr>
+<h2 id="chap_2">Функции</h2>
+<p><a href="#chap_3"> К содержанию</a></p> 
+<p>Когда мы импортировали модуль random, мы использовали функцию <strong>randint</strong>(). Функция — это блок кода, который создан для того, чтобы вызывать его в программе, обычно, более одного раза. Это 
+упрощает работу над программой и предотвращает набор одного и того же кода снова и снова. Грубо говоря, если вам нужно написать некоторый код больше, чем 1-2 раза, стоит сделать его функцией. Хотя следующие два примера очень простые, это хорошие примеры функций. Теперь мы хотим, скажем, взять два числа, сложить их, перемножить, вычесть одно из другого, затем вывести на экран эти числа и результаты одновременно. Допустим, нам нужно сделать это три раза для трёх наборов чисел. Тогда наш пример будет выглядеть как код ниже.</p>
+<pre>#silly example
+print '%d + %d = %d ' % (1,2,1+2)
+print '%d * %d = %d ' % (1,2,1*2)
+print '%d - %d = %d ' % (1,2,1-2)</pre> 
+ 
+<pre>print '\n'
+print '%d + %d = %d ' % (1,4,1+4)
+print '%d * %d = %d ' % (1,4,1*4)
+print '%d - %d = %d ' % (1,4,1-4)</pre>
+ 
+<pre>print '\n'
+print '%d + %d = %d ' % (10,5,10+5)
+print '%d * %d = %d ' % (10,5,10*5)
+print '%d - %d = %d ' % (10,5,10-5)
+print '\n'</pre> 
+<p>Здесь нужно не только набрать много текста, но и непременно возникнут ошибки как при написании, так и при внесении изменений. Вместо этого мы создадим функцию, которую назовём «<strong>DoTwo</strong>». Она будет каждый раз брать два числа, проводить математические операции и выводить результат. Начнём с ключевого слова «<strong>def</strong>», которое сигнализирует о том, что мы собираемся определить функцию. После слова «<strong>def</strong>» добавим название функции и список параметров, если таковые имеются, в круглых скобках. Эту строку необходимо завершить двоеточием. Код функции набирается с отступом. Наш улучшенный простой пример (<ins>№2</ins>) показан ниже.</p> 
+<pre>#silly example 2...still silly, but better
+def DoTwo(num1,num2): 
+	print '%d + %d = %d ' % (num1,num2,num1+num2) 
+	print '%d * %d = %d ' % (num1,num2,num1*num2) 
+	print '%d - %d = %d ' % (num1,num2,num1-num2) 
+	print '\n'
+DoTwo(1,2)
+DoTwo(1,4)
+DoTwo(10,5)</pre> 
+<p>Как видите, здесь написано меньше кода — 8 строк вместо 12. Если нам нужно будет что-либо поменять в функции, мы сможем сделать это, не создавая проблем для нашей программы. Чтобы вызвать функцию, нужно написать её имя и параметры.</p> 
+	<table width="1420" border="0">
```

Comment:
Could you please use % here, need to be more adaptive, check all  your python files

Reply:
Can you please explain a bit more in details cos I don't really get the problem here ? thx =)

Reply:
You have 1420 - as width, I have large screen and your menu to sort, you need to make pages more adaptive